.github/workflows/verify_library_generation.yaml

```diff
@@ -38,3 +38,13 @@ jobs:
       run: |
         set -x
         library_generation/test/generate_library_unit_tests.sh
+  lint:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v3
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@2.0.0
```

Comment:
@suztomo Do we have any guidelines or best practices regarding using third_party Github actions? This one has 200+ stars so there are some usages, but do we think there is any risks in using it?

Reply:
I don't think of any risk using this because it doesn't add any code to the repository.

Reply:
The underlying [ShellCheck](https://github.com/koalaman/shellcheck) is open-sourced and recommended by Google.

If we think third-party action is unreliable, I can write a commend to invoke `shellcheck` directly. WDYT?

Reply:
I think we can use the action for now and if it's broken in the future, I can always convert it to `shellcheck` command.

Reply:
It would be nice to be able to run it locally too.